pkg/performanceprofile/controller/performanceprofile_controller_hypershift.go

```diff
@@ -71,3 +102,230 @@ func (r *PerformanceProfileReconciler) hypershiftSetupWithManager(mgr ctrl.Manag
 
 	return ctrl.NewControllerManagedBy(mgr).For(&corev1.ConfigMap{}, builder.WithPredicates(p)).Complete(r)
 }
+
+func (r *PerformanceProfileReconciler) hypershiftReconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+
+	if !ntconfig.InHyperShift() {
+		return reconcile.Result{}, fmt.Errorf("Using hypershift controller configuration while not in hypershift deployment")
+	}
+
+	instance := &corev1.ConfigMap{}
+
+	//REVIEW - Should be OperatorNamespace?
+	err := r.Get(ctx, req.NamespacedName, instance)
```

Comment:
please clarify

Reply:
Namespace is correct.
Need to use ManagementClient instead.